src/Migrate/Migrate/ChangeLog.md

```diff
@@ -29,6 +29,8 @@
 * Fixed bugs in `Get-AzMigrateLocalServerReplication`
   - Added ARM id validation for input parameters
 * Enhanced Get-AzMigrateServerMigrationStatus to add support for the -Expedite parameter.
+* Fixed bugs in `Initialize-AzMigrateLocalReplicationInfrastructure`
```

Comment:
Can you please move the new entries to the top of "## Upcoming Release"? Between line 20-21. Helps prevent bad merge